### ISSUE TITLE
taxon named "A" is not matched anymore to GBIF Backbone taxon named "A", but "B"

### ISSUE THREAD

**Issue by damianooldoni** [gbif/portal-feedback#4946]
Hi 👋 

I would like to report that the taxon _Lampropeltis getula (Linnaeus, 1766)_ (taxonKey: [209460581](https://www.gbif.org/species/209460581)) from the [List of Invasive Alien Species of Union concern](https://www.gbif.org/dataset/79d65658-526c-4c78-9d24-1870d67f8439) is not anymore matched to the homonymous taxon in the GBIF Backbone (taxonKey: [9799308](https://www.gbif.org/species/9799308)), but it is now matched to _Lampropeltis getula (Blainville, 1835)_ (taxonKey: [5224480](https://www.gbif.org/species/5224480)).

This seems to me a strange bahavior, isn't? How can a taxon not being matched to its homonymous version in the Backbone?  Thanks in advance for any help.

**Comment by ManonGros** [comment]
Yes it is very odd indeed. Especially since the taxonomic status of Lampropeltis getula (Blainville, 1835) is "doubtful". @mdoering would it be possible to take a look?
